doc/getting_started/installation_mac.rst

```diff
@@ -5,143 +5,47 @@ Development Environment Setup on macOS
 
 This section describes how to set up a macOS development system.
 
```

Comment:
Can we perhaps add a reference to `host_setup` so that users that were linked to this page directly can also go back and see the full installation picture?

Reply:
Good idea, I've added this notice to the top of all three OS docs:

```
.. important::

   This section just describes OS-specific setup; it is the first step in the
   complete Zephyr :ref:`getting_started`.
```